src/Sylius/Component/Core/Promotion/Action/FixedDiscountPromotionActionCommand.php

```diff
@@ -24,16 +24,16 @@ final class FixedDiscountPromotionActionCommand extends DiscountPromotionActionC
 {
     public const TYPE = 'order_fixed_discount';
 
-    private ProportionalIntegerDistributorInterface $proportionalDistributor;
-
     private UnitsPromotionAdjustmentsApplicatorInterface $unitsPromotionAdjustmentsApplicator;
 
+    private MinimumPriceDistributorInterface $minimumPriceDistributor;
+
     public function __construct(
-        ProportionalIntegerDistributorInterface $proportionalIntegerDistributor,
-        UnitsPromotionAdjustmentsApplicatorInterface $unitsPromotionAdjustmentsApplicator
+        UnitsPromotionAdjustmentsApplicatorInterface $unitsPromotionAdjustmentsApplicator,
+        MinimumPriceDistributorInterface $minimumPriceDistributor
```

Comment:
Previous services should be left untouched, and new argument should ba nullable. If new service is not provided, then we should fallback to previous logic
```suggestion
        ProportionalIntegerDistributorInterface $proportionalIntegerDistributor,
        UnitsPromotionAdjustmentsApplicatorInterface $unitsPromotionAdjustmentsApplicator,
        ?MinimumPriceDistributorInterface $minimumPriceDistributor
```

Reply:
added nullable arguments along with spec coverage